examples/java/README.md

```diff
@@ -0,0 +1,80 @@
+# Running the example
+Several functions have been added to this plugin to give an example of how you can add your own functions to a java plugin. Evaluate script supports script evaluation without input parameters and script evaluation 
+with input parameters if it is not an aggregation and if the script can be evaluated for one row of input parameters independently of the other rows of parameters. The scripts being evaluated however are JavaScript scripts 
+since there is no "eval" function in java.
+
+Synchronous/asynchronous:
+The plugin is partly asynchronous since several method calls can be made to the plugin at the same time. But a `StreamObserver` (which sends data to and from the plugin) can only handle one call at a time, and they need 
+to be in a specific order, therefore the plugin cannot be fully asynchronous.
+
+## Prerequisites
```

Comment:
Does this have to be done by the user? Can't we have this structure from the beginning, to make it easier?